Xamarin.Forms.Platform.WP8/ButtonRenderer.cs

```diff
@@ -127,26 +137,31 @@ void UpdateContent()
 				case Button.ButtonContentLayout.ImagePosition.Top:
 					container.Orientation = Orientation.Vertical;
 					image.Margin = new WThickness(0, 0, 0, spacing);
+					container.Children.Add(image);
+					container.Children.Add(textBlock);
 					break;
 				case Button.ButtonContentLayout.ImagePosition.Bottom:
 					container.Orientation = Orientation.Vertical;
 					image.Margin = new WThickness(0, spacing, 0, 0);
+					container.Children.Add(textBlock);
+					container.Children.Add(image);
```

Comment:
At first glance it looks like duplication, but all four cases are different (the order in which you add the image and the textblock are important). If you look at the previous version you can see where I screwed this up last time by trying to pull out "duplicate" code.